packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -504,13 +565,20 @@ class MultiFrameImageStreamCompleter extends ImageStreamCompleter {
   ///
   /// Immediately starts decoding the first image frame when the codec is ready.
   ///
-  /// [codec] is a future for an initialized [ui.Codec] that will be used to
+  /// `codec` is a future for an initialized [ui.Codec] that will be used to
```

Comment:
nit: while you are in here, maybe fix this to comply with https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar.

Reply:
Done